packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -40,18 +42,29 @@ abstract class BaseMockBlobStorage
 }
 
 class DedupeStorage extends BaseMockBlobStorage {
+	private _createBlobCount: number = 0;
+	private _min_TTL: number = 24 * 60 * 60; // same as ODSP
+
+	set min_TTL(minTTL: number) {
```

Comment:
nit: since these are mocks, I think it would be simpler to just make these public rather than add setters/getters